docs/designers-developers/developers/tutorials/format-api/1-register-format.md

```diff
@@ -38,6 +40,18 @@ Then add a new file named `my-custom-format.js` with the following contents:
 	);
 } )( window.wp );
 ```
+{% ESNext %}
+```js
+const { registerFormatType } = wp.richText;
+
+registerFormatType(
+	'my-custom-format/sample-output', {
+		title: 'Sample output',
+		tagName: 'samp',
+		className: null,
+	}
+);
+```
```

Comment:
Looks like it is missing the closing `{% end %}` tag.

Reply:
Oops sorry